deny.toml

```diff
@@ -48,8 +48,6 @@ notice = "warn"
 # A list of advisory IDs to ignore. Note that ignored advisories will still
 # output a note when they are encountered.
 ignore = [
-	# yaml-rust < clap. Not feasible to upgrade and also not possible to trigger in practice.
-	"RUSTSEC-2018-0006",
```

Comment:
If it's fine to get rid of this, we should probably also do it in the `parity-bridges-common` repo